README.md

```diff
@@ -1,24 +1,41 @@
-<div align="center">
-  <p align="center">
-    <img alt="Medusa" src="https://uploads-ssl.webflow.com/61fba9f6deac297b7b22017d/62000006ce573a706c92546c_logo.svg" width="200" />
-  </p>
-  <h1>Medusa Payment Paystack</h1>
-  <p>Add support for Paystack payments in Medusa Commerce applications.</p>
-</div>
+![Medusa Paystack Plugin](https://res.cloudinary.com/femakin/image/upload/v1666371662/Frame_59841_cuqbl1.png)
```

Comment:
Love the logo :) by dragging the image onto the Readme in the GIthub online editor you can host the image directly on Github. The cloudinary link could die sometime in the future